tsconfig.dev.json

```diff
@@ -1,28 +1,5 @@
 {
-  "compilerOptions": {
```

Comment:
Don't we need the compilerOptoins anymore?

Reply:
Nope, 
You can see that I now extends our base tsconfig.json file that already contains the relevant compilerOptions
`"extends": "./tsconfig",`